enos/modules/seal_pkcs11/main.tf

```diff
@@ -95,6 +95,7 @@ module "target" {
   ami_id          = module.ec2_info.ami_ids["arm64"]["ubuntu"]["22.04"]
   cluster_tag_key = local.id
   common_tags     = var.common_tags
+  disable_selinux = false
```

Comment:
Huh, won't this make RHEL 9.3 not work with pkcs11 since we disable it for vault to start up there?

Reply:
Ah, this was from earlier work, it's not needed now. Removed.